src/operator/ignoreElements.ts

```diff
@@ -3,6 +3,14 @@ import {Operator} from '../Operator';
 import {Subscriber} from '../Subscriber';
 import {noop} from '../util/noop';
 
+/**
+ * Ignores all items emitted by the source Observable and only passes calls of onCompleted or onError.
```

Comment:
Note that in RxJS 5 jargon, `onCompleted` is `complete` and `onError` is `error`.